lib/buildinfo/osversion_openbsd.go

```diff
@@ -0,0 +1,13 @@
+// +build openbsd
+
+package buildinfo
+
+// gopsutil v3.21.3 fails to build on openbsd:
+// Error: .../go/pkg/mod/github.com/tklauser/go-sysconf@v0.3.4/sysconf_openbsd.go:22:28: undefined: unix.RLIMIT_NPROC
+// Error: .../go/pkg/mod/github.com/shirou/gopsutil/v3@v3.21.3/process/process.go:163:15: undefined: pidsWithContext
+// and so on...
+
+// GetOSVersion returns OS version, kernel and bitness
+func GetOSVersion() (osVersion, osKernel string) {
+	return "OpenBSD", ""
```

Comment:
You could call [syscall.Uname](https://pkg.go.dev/golang.org/x/sys/unix#Uname) here and return [some things out of there](https://man7.org/linux/man-pages/man2/uname.2.html).

This should work on OpenBSD I think...

Reply:
The catch is how to test it. I don't have openbsd within reach.

Reply:
We can try with virtualbox and https://app.vagrantup.com/generic/boxes/openbsd6 but there is no guarantee that resulting code doesn't panic with other actual flavors of OpenBSD.

Reply:
I think I can open an upstream issue asking to make gopsutil work on OpenBSD. This OS is not primary for rclone. We can wait.

Reply:
They already have a PR https://github.com/shirou/gopsutil/pull/993 and it's actively moving on. I've subscribed on it. We can either put this one on hold waiting for upstream or postpone dependency upgrade to a later PR.

Reply:
What do you think @ncw ?

Reply:
That sounds fine - it is a minority platform (sorry FreeBSD users) so happy to wait until that PR is merged.

Reply:
OK. Will submit another PR when https://github.com/shirou/gopsutil/pull/993 ticks for me.